src/Contexts/NavigationContext.tsx

```diff
@@ -33,6 +33,11 @@ export const NavigationContextContextProvider = ({children}: any) => {
             text: 'Test',
             route: RoutesEnum.TEST
         },
+        {
+            icon: <LogoutIcon />,
+            text: 'Login',
+            route: RoutesEnum.LOGIN
+        },
```

Comment:
NavigationContext is what we see in the side nav bar, so this is adding a side nav icon to the login page, with the text 'Login': 
<img width="233" alt="image" src="https://user-images.githubusercontent.com/126940746/224670413-9ad6ba2c-1c3a-4807-97c8-79d6c43e1095.png">

What you could do is use the 'Logout (Inactive)' text (and remove the inactive), and just set the route to RoutesEnum.LOGIN. 
I.e. we click LOGOUT -> it routes the user to the login page again
if you want to get really fancy (but you don't have to at this stage), you could add a confirmation box when they click this